.nvmrc

```diff
@@ -0,0 +1 @@
+22.8.0
```

Comment:
Is this a requirement for the stories stuff?

Reply:
No, this is to enforce a specific node version for development in the app.  We both ran into issues where are local node version was incompatible.  This addition is to reduce friction when contributing the codebase.

Reply:
Makes sense. Do we know what was failing?

Reply:
The build process would fail and there were also some strange behaviours from yarn installing incorrect package versions due to compatibility issues between node versions.